### DISCUSSION TITLE
Quick fix "Import" does not work for modifier private with access qualifier

### DISCUSSION THREAD

**Discussion by LeUser111** [scalameta/metals#6730] [General]
Hi there,

I'm making massive use of access modifiers in combination with access qualifiers (see [Scala 2 language specification](https://scala-lang.org/files/archive/spec/2.13/05-classes-and-objects.html#modifiers)) throughout my code bases.

Unfortunately, metals does not suggest imports for members with modifier _private_ and matching qualifier.

A simple example would be: 

```scala
package com.airhack.a

private[airhack] object A {
  
    val hello = "hello"

}
```

```scala
package com.airhack.b

object B {
  
    // A is missing the import, metals doesn't suggest importing A, even though the import would be valid
    val hello = A.hello

}
```

The import quick fix works perfectly fine when _A_ is public. Right now this means that I have to write a lot of imports by hand. It would be really nice if metals could suggest the imports automatically!

I'm not sure if this would be a bug or feature request and whether metals is even the right project - it might be an upstream issue.

I'd be grateful if someone could point me in the right direction!

**Comment by tgodzik** [comment]
This looks like a bug, I opened an issue based on the discussion in https://github.com/scalameta/metals/issues/6732

**Reply by LeUser111** [reply]
Thank you Tomasz! :)
